cherry-picker/package.json

```diff
@@ -12,7 +12,7 @@
   "license": "ISC",
   "dependencies": {
     "@actions/core": "^1.9.1",
-    "@actions/github": "^2.2.0",
+    "@actions/github": "^5.0.0",
```

Comment:
There is a newer version of this library, `5.1.1`. If it doesn't cause a problem, we ought to upgrade to that now.